.github/workflows/tool-tests.yml

```
@@ -1,38 +1,23 @@
-# This is a basic workflow to help you get started with Actions
+name: Tool Tests
 
-name: CI
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
 
-# Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the develop branch
   push:
-    branches: [ develop ]
-  pull_request:
-    branches: [ develop ]
+    branches: "**"
 
-  # Allows you to run this workflow manually from the Actions tab
   workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
-  # This workflow contains a single job called "build"
-  build:
-    # The type of runner that the job will run on
+  unit_test_execution:
     runs-on: ubuntu-latest
 
-    # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
-      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
       - uses: actions/checkout@v2
```

Please provide a name for a cleaner UX.
```suggestion
      - name: Check-out repository
        uses: actions/checkout@v2
```

---

Fixed